userTests/postcss/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
     "compilerOptions": {
+        "noImplicitAny": false,
```

Comment:
Just curious, where is this needed?

It's fine to have it, since we need better testing of non-strict modes, but I couldn't guess where it applied.

Reply:
:/ the double negative confused me.

I wanted the opposite.
I'll follow up with another PR to remove this.